src/Redistribution.sol

```diff
@@ -9,93 +9,70 @@ import "./Staking.sol";
 /**
  * @title Redistribution contract
  * @author The Swarm Authors
- * @dev Implements a Schelling Co-ordination game to form consensus around the Reserve Commitment hash. This takes
- * place in three phases: _commit_, _reveal_ and _claim_.
- *
- * A node, upon establishing that it _isParticipatingInUpcomingRound_, i.e. it's overlay falls within proximity order
- * of its reported depth, prepares a "reserve commitment hash" and calculates the "storage depth" using the chunks
- * it is currently stores in its reserve. These values should be the same for every node in a neighbourhood, and
- * represent the Schelling point. Each eligible node may use these values, together with a random, single use, secret
- * _revealNonce_ and their _overlay_ as the pre-image values for the obsfucated _commit_.
- *
- * Once the _commit_ round has elapsed, participating nodes must provide the values used to calculate their obsfucated
- * _commit_ hash, which, once verified for correctness and proximity to the anchor are retained in the _currentReveals_.
- * Nodes that have commited but do not reveal the correct pre-image values will have their stake "frozen" for a period
- * of rounds proportional to their reported depth.
- *
- * During the _reveal_ round, the randomness is updated after every successful reveal. Once the reveal round
- * is concluded, the _currentRoundAnchor_ is updated and users can determine if they will be eligible their overlay
- * will be eligible for the next commit phase using _isParticipatingInUpcomingRound_.
- *
- * When the _reveal_ phase has been concluded, the claim phase can begin. At this point the truth teller and winner
- * are already determined. By calling _isWinner_, an applicant node can run the relevant logic do determine if they have
- * been selected as the beneficiary of this round. When calling _claim_, the current pot from the PostageStamp contract
- * is withdrawn and transferred to the beneficiaries address. Nodes that have revealed values that differ from the truth,
- * have their stakes "frozen" for a period of rounds proportional to their reported depth.
+ * @dev The redistribution contracts allows users to create and manage postage stamp batches.
```

Comment:
your PR pulled in some random commit, remove the changes to src/Redistribution.sol

Reply:
Please remember to self review your changes/PRs